controller/github.js

```diff
@@ -120,13 +130,22 @@ function obtainAndWriteAuth({ username, password }) {
 
 module.exports.obtainAndWriteAuth = obtainAndWriteAuth;
 
-function writeUser(user) {
+function writeUser(auth) {
   deleteUser();
   console.log('Writing user. . .'.yellow);
   return new Promise(function (res, rej) {
-    ensureApplicationDirectory();
-    fsJson.saveSync(userFilePath, user);
-    res(user);
+    const cmd = getGithubID(auth.token);
+    exec(cmd, function (err, stdout, stderr) {
+      const { id, message } = JSON.parse(stdout);
```

Comment:
Could be wrong, but I don't think we can depend on `stdout` being parsable JSON. 
```suggestion
      try {
         const { id, message } = JSON.parse(stdout);
         } catch (_e) {
           // _e is the parse error, which is not important
            return rej(`Non-JSON response:\n${stdout}`)
         }
```